server/jetstream_test.go

```diff
@@ -11211,12 +11211,13 @@ func TestJetStreamOperatorAccounts(t *testing.T) {
 func TestJetStreamServerDomainBadConfig(t *testing.T) {
 	shouldFail := func(domain string) {
 		t.Helper()
-		conf := createConfFile(t, []byte(fmt.Sprintf("jetstream: {domain: \"%s\"}", domain)))
-		defer removeFile(t, conf)
-		if _, err := ProcessConfigFile(conf); err == nil {
-			t.Fatalf("Expected bad domain error, got %v", err)
+		opts := DefaultTestOptions
+		opts.JetStreamDomain = domain
+		if err := validateOptions(&opts); err == nil {
```

Comment:
This is fine, but to make sure that you are getting the error you are checking for, I usually do:
```
if err := validateOptions(&opts); err == nil || !strings.Contains(err.Error(), "Invalid domain name") {
    t.Fatalf("Expected bad domain error, got %v", err)
}
```

Reply:
Fixed.